lld/test/ELF/aarch64-build-attributes-malformed.s

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
Don't start with unneeded \n.

yaml `Content` is difficult to updated. Use assembly test with some comments, e.g.
```
% cat x.s
.section .ARM.attributes,"",%0x70000003
.long 0  // 
.long 0  //
```